### ISSUE TITLE
determine the exact parameters used for training with scdd

### ISSUE THREAD

**Issue by shivahanifi** [shivahanifi/SCDD-image-segmentation-keras#3]
The models were trained with the same default settings, rather than optimizing hyper parameters for each model. 

|Name|value|
|---|---|
|Optimizer| adaptive moment (Adam)|
| learning rate | 0.001|
|Loss function|categorical cross-entropy|

### Parameters I'm not sure about
|Name|default value|used value|Notes|
|----|---|---|---|
|steps_per_epoch|512|len(os.listdir(train_image_path)) // batch_size||
|batch_size|2|2||
|epochs|5|5|there are notes about around 600 epochs(TBC)|
